src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import {
+  Footer,
+  ButtonContainer,
+  FooterText,
+  AboutText,
+  ModalButton,
+} from './ChartFooter.style';
+import NewDialog from 'components/NewDialog/NewDialog';
+import { Link } from 'cms-content/modals';
+import { MobileOnly, DesktopOnly } from '../Shared/Shared.style';
+import ShareButtons from 'components/LocationPage/ShareButtons';
+import { Region } from 'common/regions';
+import type { MetricValues } from 'common/models/Projections';
+
+export interface AboutModalProps {
+  header: string;
+  body: string;
+  links: Link[];
+}
+
+export interface ShareButtonProps {
+  region: Region;
+  stats: MetricValues;
+  chartIdentifier: number;
+  showEmbedButton?: boolean;
+}
+
+const MetricChartFooter: React.FC<{
+  footerText: string | null;
+  disclaimer: string | null;
```

Comment:
if theres no disclaimer for the location, i think it will just be undefined, not null. so `disclaimer?: string;` would work